composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "friendsofsymfony/http-cache": "^2.3",
+        "friendsofsymfony/http-cache": "^2.5",
```

Comment:
ah, actually this is causing various problems with the lowest version build. 

lets bump the branch alias to 2.5 and increase version numbers as needed.

Reply:
I might have misunderstood here but I bumped it and it still fails :)

Reply:
that was the start, now we have to get rid of old versions of things that we no longer can support. FOSHttpCache stopped supporting symfony 2.x in 2.5. we have to adjust here as well, and can remove the lts 2 build from the matrix.

Reply:
Ah true, wouldn't it be better to update master then and I rebase? Otherwise this PR includes a lot of actually unrelated stuff :)